src/portablectl/portablectl.c

```diff
@@ -0,0 +1,1350 @@
+                        }
+
+                        v = strjoin("/usr/lib/systemd/portable/profile/", optarg, "/");
+                        if (!v)
+                                return log_oom();
+                        if (laccess(v, F_OK) < 0)
+                                return log_error_errno(errno, "Unit profile does not exist (try --profile=help for listing available unit profiles): %s", v);
+
+                        arg_profile = optarg;
+                        break;
+                }
+
+                case ARG_COPY:
+                        r = parse_boolean(optarg);
+                        if (r < 0) {
+                                if (streq(optarg, "unit"))
+                                        arg_copy_mode = COPY_UNIT;
+                                else {
+                                        log_error("Failed to parse --copy= argument: %s", optarg);
+                                        return -EINVAL;
+                                }
+                        } else
+                                arg_copy_mode = r ? COPY_YES : COPY_NO;
+
+                        break;
+
+                case ARG_CAT:
+                        arg_cat = true;
+                        break;
+
+                case ARG_RUNTIME:
+                        arg_runtime = true;
+                        break;
+
+                case 'q':
+                        arg_quiet = true;
+                        break;
+
+                case ARG_ROOT_HASH: {
+                        void *p;
+                        size_t l;
+
+                        r = unhexmem(optarg, strlen(optarg), &p, &l);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to parse root hash: %s", optarg);
+                        if (l < sizeof(sd_id128_t)) {
+                                log_error("Root hash must be at least 128bit long: %s", optarg);
+                                free(p);
+                                return -EINVAL;
+                        }
+
+                        free(arg_root_hash);
+                        arg_root_hash = p;
+                        arg_root_hash_size = l;
+                        break;
+                }
+
+                case ARG_NO_RELOAD:
+                        arg_reload = false;
+                        break;
+
+                case '?':
+                        return -EINVAL;
+
+                default:
+                        assert_not_reached("Unhandled option");
+                }
+        }
+
+        return 1;
+}
+
+int main(int argc, char *argv[]) {
+
+        static const Verb verbs[] = {
+                { "help",    VERB_ANY, VERB_ANY, 0,                 help    },
+                { "attach",  2,        VERB_ANY, VERB_MUST_BE_ROOT, attach  },
+                { "detach",  2,        2,        VERB_MUST_BE_ROOT, detach  },
+                { "inspect", 2,        VERB_ANY, 0,                 inspect },
```

Comment:
Do you think `busctl introspect` should be changed to 'inspect'? It is completely off topic, sorry.

Reply:
so, we discussed this before, and there is (was?) a todo list item about that.

I think "Introspect" is the word usually used in OOM context, and i figure dbus qualifies for that, hence I think we should probably stick to "introspect" for that. otoh "inspect" is a ncier word in other contexts i figure

Reply:
Oh, I found the TODO entry. Thanks.